apps/etterlatte-statistikk/src/test/kotlin/statistikk/domain/BeregningTest.kt

```diff
@@ -0,0 +1,104 @@
+package statistikk.domain
```

Comment:
nav.no?

Reply:
hmm ja ser at det er litt dust bruk av pakker i testene for statistikk. Fikser det i en egen pr!